src/wwwroot/startupscripts/startagent-mac.sh

```diff
@@ -2,6 +2,15 @@
 
 workspace_path=$1
 
+# Try to clean workspace 
```

Comment:
this is the wrong script.  We don't use this one, we only use hosted macs

Reply:
Moving to startagent-linux.sh